lib/services/irc.rb

```diff
@@ -58,10 +58,11 @@ def send_messages(messages)
     rooms   = rooms.gsub(",", " ").split(" ").map{|room| room[0].chr == '#' ? room : "##{room}"}
     botname = data['nick'].to_s.empty? ? "GitHub#{rand(200)}" : data['nick'][0..16]
     command = config_boolean_true?('notice') ? 'NOTICE' : 'PRIVMSG'
+    geckos  = (defined?(sender) && defined?(repo)) ? "GitHub IRCBot - #{sender.login}/#{repo.name}" : "Github IRCBot"
```

Comment:
@Zarthus: Similar to the approach you proposed in https://github.com/github/github-services/issues/1016#issue-55607375, I recommend extracting this logic into an intention-revealing method.

What's a good name for the method? Perhaps `irc_realname`?